crosswalks-sssom/distutils/distutils-codemeta-mappings.csv

```diff
@@ -0,0 +1,32 @@
+source_term,codemeta_term,type_relation,combined_mappings,comments
+name,name,exact_match,,
+description,description,exact_match,,
+summary,headline,more_specific_than,,
+version,version,exact_match,,
+classifiers,licence,more_generic_than,,Deprecated property. Use classifiers['License']. Example: License :: OSI Approved :: MIT License
+license,license,exact_match,,If the license in Codemeta is a URL
+license,license/name,close_match,,If the license in Codemeta is a CreativeWork
+license_expression,license/name,close_match,,Since version 2.4. It only applies to CreativeWork. Example: MIT AND (Apache-2.0 OR BSD-2-Clause)
+author,author/name,exact_match,,It may include the email in the string.
+author_email,author/email,exact_match,,It may include the name in the string.
+maintainer,maintainer/name,exact_match,,It may include the email in the string.
+maintainer_email,maintainer/exact,close_match,,It may include the name in the string.
+keywords,keywords,exact_match,,
+version,softwareVersion,exact_match,,
+requires_dist,softwareRequirements,close_match,,"Example: bs4<0.0.2,>=0.0.1"
+classifiers,softwareRequirements,more_specific_than,,"Example: Programming Language :: Python :: 3. It appears more python version as list.Use classifiers['programming Language']"
```

Comment:
As discussed today, the explanation should be improved. It is not classifiers['Programming Language'], but classifiers[i].contains("Programming Language"). The example is correct, it's just that it could be misleading.
classifiers is a list of text strings. If it contains License then the mapping is correct.